src/validation/input-params.ts

```diff
@@ -1,32 +1,20 @@
+import type { AnySchema, ErrorObject, ValidateFunction } from 'ajv'
+import { getValidator } from './schema-cache'
+
 import { Overrides, ReservedInputParameterNames, hasRepeatedValues } from '../util'
 import { AdapterError } from './error'
 
-/**
- * Type for the overrides object that is either hardcoded in the adapter,
- * or present in the incoming request body
- */
+/* -------------------------------------------------------------------------- */
+/*                               TYPE HELPERS                                 */
+/* -------------------------------------------------------------------------- */
+
```

Comment:
I would DEFINITELY not remove all these comments haha, I added them with a lot of love and I'm sure our AI overlords appreciate them even if they hallucinate to remove them